clients/js/test/legacy/buy.test.ts

```diff
@@ -218,11 +261,34 @@ test('buy from NFT pool, skip non-rent-exempt creators', async (t) => {
     (tx) => signAndSendTransaction(client, tx)
   );
 
-  await testBuyNft(t, legacyTest, {
+  // Ensure creators are now verified.
+  const md = await fetchMetadata(client.rpc, legacyTest.nft.metadata);
+
+  if (isSome(md.data.creators)) {
+    creators = md.data.creators.value;
+  } else {
+    creators = [];
+  }
+
+  t.assert(creators.every((c) => c.verified));
+
+  const creatorStartingBalances = await Promise.all(
+    creatorSigners.map((c) => getBalance(client, c.address))
+  );
+
+  await testBuy(t, legacyTest, {
     brokerPayments: false,
-    creators,
+    creators, // pass in the verified creators
+    checkCreatorBalances: false,
+    pNft: true,
   });
 
+  // First three creators should have a higher balance.
```

Comment:
why are we not checking exact creator balance diffs depending on their share?

Reply:
Test fatigue. Fixed.